lstchain/scripts/lstchain_data_create_time_calibration_file.py

```diff
@@ -57,7 +56,39 @@
                     help='Path to drs4 pedestal file ',
                     default=None
                     )
+parser.add_argument('--ucts-t0-dragon', action='store', type=float,
```

Comment:
Yes, ints. Default should be None, not nan

Reply:
> Yes, ints. Default should be None, not nan

I get the error below when trying to run r0_to_dl1 (which worked on this branch a few commits ago). Is it perhaps related?

```
  File "/nfs/cta-ifae/moralejo/miniconda3/envs/lst-dev/lib/python3.7/site-packages/traitlets/traitlets.py", line 2040, in validate
    self.error(obj, value)
  File "/nfs/cta-ifae/moralejo/miniconda3/envs/lst-dev/lib/python3.7/site-packages/traitlets/traitlets.py", line 681, in error
    raise TraitError(value, info or self.info(), self)
traitlets.traitlets.TraitError: (1.5953039761947725e+18, 'an int', <ctapipe_io_lst.event_time.Int object at 0x7fcc88706e50>)
```

Reply:
Yes, please note that the tests are all falling right now

Reply:
The new version of `ctapipe_io_lst` will always read the night summary columns as int now and will use a masked array if there are any nans. 

So this command line tool will need to pass the correct options depending on whether or not these counters are available.

Currently the `EventTimeCalculator` will only work if ucts and tib are available. I am working on enabling to relaxing this to only requiring that UCTS is available and making tib optional. (Is there any reason to use TIB over Dragon? Dragon is always available, right?).

The problem remaining is how to handle data with missing UCTS, since no absolute timestamps are possible to my knowledge without UCTS. 

Currently the `LSTEventSource.fill_timestamp` traitlet can be set to false, which will result in all timestamps being the same invalid time marker. 

If code does not need absolute times, that would be an option.

Reply:
I can work on this stuff, but I don't know what to do when.

Please give input here: https://github.com/cta-observatory/ctapipe_io_lst/issues/71